### PR TITLE
Improve documentation coverage for Phobert 

### LINKED ISSUE #9035: Improve coverage of the documentation
Currently, some public classes are not documented anywhere because we didn't create the corresponding doc pages. Those missing pages are:

- Benchmark classes
- Bert Japanese
- Data collators

If someone feels like working on one of those, please tag yourself with a comment on this issue. Once the objects are properly documented, they can be removed from the `SHOULD_BE_DOCUMENTED` constant in [this file](https://github.com/huggingface/transformers/blob/1310e1a758edc8e89ec363db76863c771fbeb1de/utils/check_repo.py#L374).

### PR DESCRIPTION
**Pull request by Qbiwan**
# What does this PR do?

Fixes #9035

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors which may be interested in your PR.

### CODE REVIEW

**Review by LysandreJik — APPROVED** [review]
LGTM, thanks @Qbiwan!

**Comment by LysandreJik** [comment]
Can't pin you for review @sgugger, so tagging you!

File: docs/source/model_doc/phobert.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  import torch
+  from transformers import AutoModel, AutoTokenizer
+
+  phobert = AutoModel.from_pretrained("vinai/phobert-base")
+
+  # For transformers v4.x+: 
+  tokenizer = AutoTokenizer.from_pretrained("vinai/phobert-base", use_fast=False)
+
+  # For transformers v3.x: 
+  # tokenizer = AutoTokenizer.from_pretrained("vinai/phobert-base")
```
Comment (sgugger) [inline docs/source/model_doc/phobert.rst:43]: There is no need to differentiate the code. In v4+, when a fast tokenizer is not available, the library will use the slow one. So just `tokenizer = AutoTokenizer.from_pretrained("vinai/phobert-base")` is good enough!

**Review by sgugger — APPROVED** [review]
Thanks for adding this!

File: docs/source/model_doc/phobert.rst (hunk 1 above)
Comment (Qbiwan) [reply docs/source/model_doc/phobert.rst]: reduced to a single line `tokenizer = AutoTokenizer.from_pretrained("vinai/phobert-base")`

**Pull request merged**